lib/capistrano/scm/git.rb

```diff
@@ -44,6 +44,9 @@ def clone_repo
   end
 
   def update_mirror
+    # Update the origin URL if necessary.
+    git :remote, "set-url", "origin", repo_url
```

Comment:
I looked to see if `git-remote set-url` had a long or short history, to see if we needed to address `git` version requirements at all. Spoiler Alert: it seems to have been around basically for **at least** [three years, since `1.8.2.3`][forever], so all fine.

This looks like a simple enough change, I'd be glad to have it in. 

[forever]: https://git-scm.com/docs/git-remote/1.8.2.3